<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditSensor</class>
 <widget class="QDialog" name="DialogEditSensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Sensor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowminimumheight="0,0,0,0,0,23,23,23">
     <item row="2" column="3">
      <widget class="QComboBox" name="m_cbxMipiFreq">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="BoolFrame" name="m_grpPwdn">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="BoolFrame" name="m_grpVsyncPol">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Mipi Freq(MHz):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QComboBox" name="m_cbxImageMode"/>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="m_edtSchemeName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Scheme Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="BoolFrame" name="m_grpPclkPol">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="m_spinMipiLanes">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>MIPI Lanes:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Interface Type:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="BoolFrame" name="m_grpDataPol">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="BoolFrame" name="m_grpHsyncPol">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="BoolFrame" name="m_grpReset">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>PCLK Pol:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>HSYNC Pol:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>DATA Pol:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>PWDN:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>VSYNC Pol:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>RESET:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Image Mode:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="m_cbxInterface"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>MCLK(MHz):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Image Format:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="m_cbxImageFormat"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Pixel Width:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Pixel Height:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="m_edtPixelWidth"/>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="m_edtPixelHeight"/>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="m_cbxMclk">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_grpVoltage">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Full Mode Registers:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Otp Init Registers:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Exposure Registers:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3">
       <widget class="QPlainTextEdit" name="m_edtFullModeRegisters"/>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="m_edtOtpInitRegisters"/>
      </item>
      <item row="1" column="2">
       <widget class="QPlainTextEdit" name="m_edtExposureRegisters"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Otp Algorithm Parameters:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Gain Registers:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPlainTextEdit" name="m_edtOtpAlgParams"/>
      </item>
      <item row="3" column="2">
       <widget class="QPlainTextEdit" name="m_edtGainRegisters"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>100</number>
     </property>
     <property name="leftMargin">
      <number>200</number>
     </property>
     <property name="rightMargin">
      <number>200</number>
     </property>
     <item>
      <widget class="QPushButton" name="m_btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BoolFrame</class>
   <extends>QFrame</extends>
   <header>ui/BoolFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_edtSchemeName</tabstop>
  <tabstop>m_btnOK</tabstop>
  <tabstop>m_btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
